Check for libdc1394 on Fedora 27

Awesome work on PlaneteryImager. Just wanted to suggest something:

    sudo dnf install \
        boost-devel \
        CCfits-devel \
        cmake \
        gcc-c++ \
        git \
        libusb-devel \
        opencv-devel \
        qt5-qtdeclarative-devel

    git clone --recursive https://github.com/GuLinux/PlanetaryImager.git
    cd PlanetaryImager && mkdir build && cd build
    cmake .. -DCMAKE_INSTALL_PREFIX=/usr
    make all && sudo make install

[...]

    [ 97%] Built target iidc_autogen
    Scanning dependencies of target iidc
    [ 98%] Building CXX object src/drivers/iidc/CMakeFiles/iidc.dir/iidc_driver.cpp.o
    /PlanetaryImager/src/drivers/iidc/iidc_driver.cpp:20:10: fatal error: dc1394/dc1394.h: No such file or directory
     #include <dc1394/dc1394.h>
          ^~~~~~~~~~~~~~~~~
    compilation terminated.
    make[2]: *** [src/drivers/iidc/CMakeFiles/iidc.dir/build.make:63: src/drivers/iidc/CMakeFiles/iidc.dir/iidc_driver.cpp.o] Error 1
    make[1]: *** [CMakeFiles/Makefile2:997: src/drivers/iidc/CMakeFiles/iidc.dir/all] Error 2
    make: *** [Makefile:163: all] Error 2


Fix:

    sudo dnf install libdc1394-devel

Maybe the cmake step should check for dc1394 ?

--- a/src/drivers/iidc/CMakeLists.txt
+++ b/src/drivers/iidc/CMakeLists.txt
@@ -1,3 +1,10 @@
 file(GLOB iidc_SRCS *.cpp)
 
 add_driver(NAME iidc OS Linux Darwin DEFAULT_ON SRCS ${iidc_SRCS})
+
+if(BUILD_DRIVER_iidc)
+    pkg_check_modules(LIBDC1394 libdc1394-2 REQUIRED)
+    include_directories(${LIBDC1394_INCLUDEDIR})
+    link_directories(${LIBDC1394_LIBRARY_DIRS})
+    target_link_libraries(iidc ${LIBDC1394_LIBRARIES})
+endif()

